sponser_dashboard.html

```diff
@@ -4,194 +4,9 @@
-            width: 46px;
-            font-size: 20px;
-            border: 1px solid #000000;
-        }
-        #fair-heading{
-            font-size: 25px;
-            margin-left: 16%;
-            font-family: Avenir;
-            display: inline-block;
-        }
-        ul{
-            list-style-type: none;
-        }
-        li {
-            float: left;
-            margin: 0px 30px 0px 0px;
-            font-size: 16px;
-            font-family: Avenir;
-        }
-
-        li a{
-            display: block;
-            color: black;
-            text-align: center;
-            padding: 14px 16px;
-            font-weight: bold;
-            text-decoration: none;
-            font-family: Avenir;
-        }
-        li a:hover{
-            border-bottom: 3px solid blue;
-            color: blue;
-        }
-        #box-container{
-            background: #ffffff;
-            box-shadow:0px 6px 10px #6596FF26;
-            border-radius: 12px;
-            width: 1066px;
-            height: 500px;
-            margin-left: 250px;
-            display: inline-block;
-            margin-top: 10px;
-        }
-        #sponsor-heading{
-            margin-left: 30px;
-            padding: 20px;
-            text-align: left;
-            font: normal normal 900 25px Avenir;
-            letter-spacing: 0px;
-            color: #1B1B1B;
-        }
-        #sponsor-button{
-            width: 170px;
-            color: white;
-            border: 2px solid #79D733;
-            text-align: center;
-            float: right;
-            height: 50px;
-            margin-top: 30px;
-            font-size: 16px;
-            font: normal normal 900 16px/48px Avenir;
-            letter-spacing: 0.8px;
-            background: #79D733;
-            margin-right: 25px;
-            text-decoration: none;
-            border-radius: 10px;
-        }
-        table {
-            margin-left: 0px;
-            text-align: center;
-            font-family: Avenir;
-        }
-        th{
-            background: rgb(238, 235, 235);
-            padding: 15px;
-            opacity: 0.9;
-        }
-        td {
-            font-family: Avenir;
-        }
-        table, th, td{
-            border-top: 1px solid rgb(195, 190, 190);
-            border-collapse: collapse;
-        }
-        .status-active{
-            background: #79D733;
-            border-radius: 10px;
-            text-align: center;
-            padding: 5px 20px 5px 20px;
-        }
-        .status-deactive{
-            background: #BECAD6;
-            border-radius: 10px;
-            text-align: center;
-            padding: 5px 20px 5px 20px;
-        }
-        .options{
-            border: 1px solid #9e9898;
-            border-radius: 8px;
-            text-decoration: none;
-            padding: 5px 10px 5px 10px;
-        }
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="sponsor-dashboard-style.css">
```

Comment:
We don't need to add `style` in file name,
better way is to create folder for css